### ISSUE TITLE
Notepad: Pressing NVDA+B or attempting to use object navigation when pressing Alt+F4 after typing something results in dialog text repeating

### ISSUE THREAD

**Issue by josephsl** [nvaccess/nvda#6972]
Hi,
Several users in Korea and I can confirm the following:

In Notepad, after typing something and pressing Alt+F4, pressing NVDA+B or attempting to use object navigation results in dialog text repeating.

STR:

1. Open Notepad.
2. Type something.
3. Press Alt+F4. From the resulting dialog, press NVDA+B or try using object navigation.

Expected: NVDA lets you navigate to the rest of the dialog via object navigation, and dialog text is announced once if NVDA+B is pressed.
Actual: Dialog text is repeated or one would find oneself going around in circles when attempting to use object navigation.

Thanks.

**Comment by PratikP1** [comment]
Not sure whether your report was limited to Korean, but when Language is set to U.S. English, both object navigation and alt+b skip the “Don’t save” button while reading. Only tab takes the user to the skipped button.

**Comment by josephsl** [comment]
Hi, I tested in English locale and can confirm the fact that pressing TAB will let you go to “don’t save”, not via object navigation. Thanks.

**Comment by ehollig** [comment]
I can confirm this issue using object navigation or NVDA+B with the above STR using English in notepad. Resulting in NVDA either repeating the information in the case of NVDA+B, or going in circles using object navigation. Using NVDA Master 14328

**Comment by Brian1Gaff** [comment]
Yes this issue on nvda/b has affected a lot of software. Its not a command I 
use a lot, and if that is all it does then OK, but are we also saying this 
has other effects, ones I've not seen?


Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.

**Comment by Adriani90** [comment]
This issue is still reproducible in NVDA 2019.1 and Notepad on Windows 10 1809 Update.
1. Type something in the editor
2. Press al+f4
3. Press nvda+b
Actual: NVDA repeats the dialog without stopping until you press a key to interupt the speech
Expected: NVDA should read the dialog only once.

4. Type something in the editor
5. Press alt+f4
6. With laptop keyboard layout, press nvda+shift+right arrow multiple times
Actual: Object naigation gets stuck and it dows not move on.
Expected: Object navigation should move on and should report (no further object).

**Comment by dnz3d4c** [comment]
I want to know why this is happening. I hope it was fixed soon.
